apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -620,4 +621,13 @@ defmodule SiteWeb.TripPlanView do
       highest_one_way_fare + acc
     end)
   end
+
+  @spec monthly_pass(Fare.t()) :: String.t()
+  def monthly_pass(fare) do
+    "#{zone(fare)}#{Format.media(fare)}: #{Format.price(fare)}"
+  end
+
+  @spec zone(Fare.t()) :: String.t()
+  defp zone(%Fare{mode: :commuter_rail, name: {:zone, zone}}), do: "Commuter Rail Zone #{zone} "
+  defp zone(_), do: ""
```

Comment:
Consider adding these formatting functions to `Fares.Format` (and perhaps leverage some more of its functionality)

Reply:
I think this got simplified and more specialized with a recent change. At this point, we just want to add "Commuter Rail" to the beginning, but not do anything for any other modes.